src/app/_components/Card/CardSpeсialist/ContactsListItem.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import P from 'prop-types';
+import Link from 'next/link';
+import { ListTruncator } from '@components';
+import { specialistContactPropType } from '@/app/_components/Card/CardSpeсialist/prop-types';
+import { cn } from '@/utils/cn';
+
+function ContactItem({ type, content, href, className }) {
+  return (
+    <>
+      {type === 'link' && (
```

Comment:
Putting values ​​into variables is good practice and improves code readability.
Consider creating some constants like:
`typesOfContact: { link: 'link', text: 'text' }`
And then reuse them in all necessary places.

Reply:
Thanks for comment! Removed `type` since login of type definition is moved into `ContactItem`.